service-api/src/main/java/greencity/constant/ValidationConstant.java

```diff
@@ -9,7 +9,7 @@ public class ValidationConstant {
     public static final String CITY_EN_REGEXP = "^([A-Z][a-z]{0,39}[ʼ'`ʹ]?[a-z]{1,39}($|[ -](?=[A-Z]))){1,10}$";
     public static final String CH_EN = "[A-Za-z\\s-ʼ'`ʹ,.]";
     public static final String CH_UA = "[ЁёІіЇїҐґЄєА-Яа-я\\s-ʼ'`ʹ,.]";
-    public static final String CH_NUM = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z0-9\\s-.]";
+    public static final String CH_NUM = "[ЁёІіЇїҐґЄєА-Яа-яA-Za-z0-9-/\\s-.]";
```

Comment:
should we add a length limit?

Reply:
why did you add `-`?
The regex already contains it

Reply:
What does `s` in this regex mean?
If it should be `whitespace character` then `\` should be added before
https://www.regular-expressions.info/shorthand.html
In this regex, it means `s` and could be removed

Reply:
`@Length(max = 5)` annotation is redundant if you add `{1,5}` to the regex

Reply:
fixed